1682/GasPoweredHALE.py

```diff
@@ -0,0 +1,31 @@
+from gas_hale import GasPoweredHALE
+import gpkit
+import numpy as np
+gpkit.settings['latex_modelname'] = False
```

Comment:
no need to do `gpkit.settings['latex_modelname'] = False` in a `.py` file. This is done for ipython notebooks for clearer model latex printing when there is no model composition happening.